api/resources_portal/views/attachment.py

```diff
@@ -96,13 +95,6 @@ def get_permissions(self):
     # because we want the path to have its id in it.
     @transaction.atomic
     def create(self, request, *args, **kwargs):
-        if not (
-            MaterialRequest.objects.filter(requester=request.user, is_active=True).exists()
-            or user_has_perm_on_active_material_request(request.user, "approve_requests")
```

Comment:
These three places are the only places this function gets used right? It seems like you should be able to delete it then.

Reply:
Good call. Deleted.